posts/Hello World.md

```diff
@@ -1,19 +1,127 @@
 ---
-title: "Hello Euri!"
+title: "Hello World!"
 date: 2022-11-09T01:10:40+01:00
 draft: false
 tags:
-    - Hello Euri!
+    - Hello World!
 comments: false
 toc: false
 featured: true
 ---
+<!--
+# Look this cool title!
+## Other
+### AAAA
+#### tester
 
-# Hello Euri!
+This is my first time using Hugo! This is a `monospaced text`
+
+---
+
+This is a section break with a box to highlight content like commands!
+```
+pixbuf_width / allocated_width = pixbuf_height / allocated_height
+```
+---
+In this section, I can write links like google home page: [google](https://www.google.es)
+
+---
+Now, look this cute puppy:
+
+![image alt text](/puppy.png)
+<!-- Comentario -->	
+
+
+<!--
+---
+
+Bloque de código en CSS:
+
+```css
+.custom-menu-tucked .custom-menu-screen {
+    -webkit-transform: translateY(-44px);
+    -moz-transform: translateY(-44px);
+    -ms-transform: translateY(-44px);
+    transform: translateY(-44px);
+}
+/* Retoques*/
+
+.pure-menu {
+    background-color: #FE0809;
+    color: white;
+    width: 100%;
+    position: sticky;
+}
+
+.pure-menu a {
+    color: white;
+}
+
+.pure-menu a:hover not(:first-child){
+    background-color: green;/*#D40A0B;*/
+}  
+```
+-->
+# Hello World!
 Welcome to our new IT site! Here we are going to post our projects. What is better beginning 
 than releasing a website. We have three servers, one is reserved to important things, other
 is where we locate all our projects and there is another one which is the f****** jungle.
 And, guess what? For this moment, we are going to be allocated there, in lab server. Maybe, if everything
-is OK, we migrate to a normal server. 
+is OK, we migrate to a normal server. All commands named in this blog is refered to Linux SO´s.
+
+How is suppose I´m goint to do that? Well, keep reading.
```

Comment:
How am I going to do that?

or

How is it supposed I am going to do that?

Reply:
Both are possible, no?

Reply:
A que no sé inglés y las he mezclado